Transformer/LuceneToBddTransformerInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace OpenOrchestra\Transformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * Interface LuceneToBddTransformerInterface
+ */
+interface LuceneToBddTransformerInterface extends DataTransformerInterface
+{
+    /**
+     * @param string $field
+     */
+    public function setField($field);
+
+    /**
+     * @param string $condition
+     *
+     * @return mixed
+     */
+    public function transform($condition);
+
+    /**
+     * @param array $value
+     *
+     * @return array
+     */
+    function reverseTransform($value);
+
+    /**
+     * @param string $condition
+     *
+     * @return mixed
+     */
+    public function reverseTransformField($condition);
```

Comment:
It seems this method is only used internally to the transformer, maybe you could put it protected and forget it in the interface.